res/layout/call_log_fragment.xml

```diff
@@ -19,7 +19,7 @@
     android:layout_width="match_parent"
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit